emulator/index.js

```diff
@@ -0,0 +1,32 @@
+// Require Mock Mobile
```

Comment:
Please exclude the emulator directory from this PR since they are, as you put it, a WIP. WIPs should live in the developer branch, not the story branch.